src/helpers/messages.js

```diff
@@ -30,7 +30,7 @@ import statsHelper from 'taoQtiTest/runner/helpers/stats';
  * @param {Boolean} sync - flag for sync the unanswered stats in exit message and the unanswered stats in the toolbox
  * @returns {String} Returns the message text
  */
-function getExitMessage(message, scope, runner, sync) {
+function getExitMessage(scope, runner, message, sync) {
```

Comment:
Why have you decided to change the public interface of the method?

Now it's technically a breaking change, with a risk of the method being used by some other part of the codebase.

Reply:
I change all affected plugins that call this function. The most common case is not pass any message because handle inside all the options.

But when I have all messages (https://oat-sa.atlassian.net/browse/BSA-210) or maybe with implementation of new test runner, this will radical change. For now, deliver this, in this week, is the priority